index.js

```diff
@@ -46,11 +46,11 @@ module.exports = (params) => {
         enableSessionTables: true, // whether to generate the sessions and pageview_with_funnels tables
         enableMonitoring: true, // whether to send summary monitoring data to the monitoring.pipeline_snapshots table in the cross-service GCP project
         urlRegex: null, // re-2 formatted regular expression to use to identify whether a URL is this service's own URL or an external one. If your service only has one domain name set this to 'www.yourdomainname.gov.uk' (without the protocol). If you have more than one use something like '(?i)(www.domain1.gov.uk|www.domain2.gov.uk|www.domain3.gov.uk)'
-        socialRefererDomainRegex: "(?i)(facebook|twitter|^t.co|linkedin|youtube|pinterest|whatsapp|tumblr|reddit)", // re-2 formatted regular expression to use to work out whether an HTTP referer is a social media site
-        searchEngineRefererDomainRegex: "(?i)(google|bing|yahoo|aol|ask.co|baidu|duckduckgo|dogpile|ecosia|exalead|gigablast|hotbot|lycos|metacrawler|mojeek|qwant|searx|swisscows|webcrawler|yandex|yippy)", // re-2 formatted regular expression to use to work out whether an HTTP referer is a search enginer (regardless of whether paid or organic)
+        socialRefererDomainRegex: "(?i)(facebook|twitter|^t.co|linkedin|youtube|pinterest|quora|wechat|weibo|whatsapp|tumblr|reddit)", // re-2 formatted regular expression to use to work out whether an HTTP referer is a social media site
+        searchEngineRefererDomainRegex: "(?i)(google|bing|yahoo|aol|ask.co|baidu|duckduckgo|dogpile|ecosia|entireweb|exalead|gigablast|hotbot|info.com|lycos|metacrawler|mojeek|qwant|searx|startpage|swisscows|webcrawler|yandex|yippy)", // re-2 formatted regular expression to use to work out whether an HTTP referer is a search enginer (regardless of whether paid or organic)
         funnelDepth: 10, // Number of steps forwards/backwards to analyse in funnels - higher allows deeper analysis, lower reduces CPU usage
         requestPathGroupingRegex: '[0-9a-zA-Z]*[0-9][0-9a-zA-Z]*', // re2-formatted regular expression to replace with the string 'UID' when grouping request paths
-        attributionParameters: ['utm_source', 'utm_campaign', 'utm_medium', 'utm_content', 'gclid', 'gcsrc'], // list of parameters to extract from the request_query array of structs at the beginning of funnels
+        attributionParameters: ['utm_source', 'utm_campaign', 'utm_medium', 'utm_content', 'gclid', 'gcsrc','fbclid','dclid','gad_source','ds_rl','gbraid','msclkid','sccid'], // list of parameters to extract from the request_query array of structs at the beginning of funnels
```

Comment:
@AndrewPetrie1 which services have incoming events that use these parameters and which do not? Each one adds processing cost/time and creates confusion if it's a blank field, so I think we should only be listing fields here which are going to be useful for most services. If they have service specific ones then they can always set their own value of ```attributionParameters```.

Reply:
All of those events relate to actual incoming events, i checked against GiT, Apply and AQTS and excluded any others I didn't see. I didn't check the frequency. 

fbclid: A Facebook click identifier automatically appended to URLs shared via Facebook ads or organic posts.
dclid: A display click identifier for Google Display Network campaigns.
id: Could represent a generic or custom tracking parameter for specific use cases.
gad_source: Likely refers to Google Ads source or traffic attribution details.
ds_rl: DoubleClick's tracking parameter for display ads, showing remarketing list identifiers.
gbraid: Google’s parameter to track app conversions for iOS and other environments.
msclkid: Microsoft Ads click identifier for tracking ad performance in Bing Ads campaigns.
ScCid: A tracking parameter often used in Salesforce or similar systems for campaign or lead tracking.

Reply:
Thanks @AndrewPetrie1 , I think please can we miss off id if we don't really know what it is? It has such strong connotations that I'd worry people confuse it for something else!

Also does DfE use Salesforce for anything? If not let's miss that one off. It's probably just random bot clicks or something unless there's actually some meaningful data in it.

Reply:
Also the presence of these parameters should feed into the ```medium``` CASE statement elsewhere. For example if there is a dclid then the medium should probably be "PPC" or something.

Reply:
@stevenleggdfe suggested changes made.